v3/plugins/vuejs/vetur/0.26.1/meta.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v2
+publisher: vuejs
+name: vetur
+version: 0.26.1
+type: VS Code extension
+displayName: Vetur
+title: Rich Vue.js support
+description: Vue tooling for VS Code
+icon: https://upload.wikimedia.org/wikipedia/commons/9/95/Vue.js_Logo_2.svg
+repository: https://github.com/vuejs/vetur.git
+category: Language
+firstPublicationDate: "2016-11-06"
```

Comment:
date should be fixed

Reply:
![image](https://user-images.githubusercontent.com/407397/91320441-e5cf5380-e793-11ea-9ee8-3a57c9759174.png)
it's the date of the first release of the extensions repo, am I taking the wrong source as value for this field?

Reply:
I believe that date should match the date (roughly) the extension was published in our plugin registry (Che), not the extension's publish date.